templates/adversaries.html

```diff
@@ -742,6 +741,10 @@ <h3>Create a profile</h3>
                     return apiV2('GET', '/api/v2/abilities');
                 }).then((abilities) => {
                     this.abilities = abilities;
+                    this.adversaries.forEach((adversary) => {
+                        adversary.tactics = [...new Set(adversary.atomic_ordering.map((ability_id) => (this.abilities.find((ability) => ability.ability_id === ability_id).tactic)))];
```

Comment:
Good catch! I wasn't able to recreate this bug on Chrome or FF, but I think the bug is in my use of the map() function instead of filter() here, so let me know if that still errors out for you.